e2e/pagination/pagination.e2e-spec.ts

```diff
@@ -5,6 +5,7 @@ interface returnState {
 	rootClasses: string[];
 	disabled: string | null;
 	pages: string[];
+	hrefs: string[];
```

Comment:
The hrefs for the boundaries are separated, so I would add some properties for them, to be included in the different tests

Reply:
indeed